extensions/sql-database-projects/src/dialogs/addDatabaseReferenceQuickpick.ts

```diff
@@ -116,16 +116,31 @@ async function addSystemDatabaseReference(project: Project): Promise<ISystemData
 
 	const selectedSystemDb = await vscode.window.showQuickPick(
 		getSystemDbOptions(project),
-		{ title: constants.systemDatabase, ignoreFocusOut: true, });
+		{ title: constants.systemDatabase, ignoreFocusOut: true });
 	if (!selectedSystemDb) {
 		// User cancelled
 		return undefined;
 	}
 
-	// 3. Prompt DB name
+	// 3 Prompt for Reference Type if it's an SDK-style project
```

Comment:
Might be nice to have this in a helper function like the the other steps in this flow for readability.

Reply:
moved this out to a helper function